misc/services/camel/de-metas-camel-externalsystems/de-metas-camel-scriptedadapter/src/main/java/de/metas/camel/externalsystems/scriptedadapter/convertmsg/to_mf/ScriptedImportConversionRestAPIRouteBuilder.java

```diff
@@ -0,0 +1,354 @@
+
+	@Override
+	public void configure()
+	{
+		CamelRouteUtil.setupProperties(getContext());
+
+		javaScriptRepo = new JavaScriptRepo(getContext().resolvePropertyPlaceholders("{{" + PROPERTY_SCRIPTING_REPO_BASE_DIR + "}}"));
+
+		//@formatter:off
+		errorHandler(defaultErrorHandler());
+		onException(Exception.class)
+				.to(direct(MF_ERROR_ROUTE_ID));
+
+		// enable http-endpoint
+		from(direct(ENABLE_RESOURCE_ROUTE_ID))
+				.routeId(ENABLE_RESOURCE_ROUTE_ID)
+				.log("Route invoked!")
+				.process(this::prepareRestAPIContext)
+				.process(this::attachAuthenticateRequest).id(ENABLE_RESOURCE_ATTACH_AUTHENTICATE_REQ_PROCESSOR_ID)
+				.to(direct(REST_API_AUTHENTICATE_TOKEN))
+				.process(this::enableRestAPIProcessor).id(ENABLE_RESOURCE_ROUTE_PROCESSOR_ID)
+				.process(exchange -> this.prepareExternalStatusCreateRequest(exchange, JsonExternalStatus.Active)).id(ENABLE_PREPARE_EXTERNAL_STATUS_CREATE_REQ_PROCESSOR_ID)
+				.to("{{" + ExternalSystemCamelConstants.MF_CREATE_EXTERNAL_SYSTEM_STATUS_V2_CAMEL_URI + "}}")
+				.end();
+
+		rest("/interchange/import")
+				.post("*")
+				.to("direct:" + REST_API_ROUTE_ID);
+
+		from("direct:" + REST_API_ROUTE_ID)
+				.routeId(REST_API_ROUTE_ID)
+				.log("Route invoked!")
+				.doTry()
+					.process(this::restAPIProcessor)
+					.toD("direct:${exchangeProperty.endpointName}")
+					.process(this::prepareResponse)
+					.marshal(setupJacksonDataFormatFor(getContext(), Object.class))
+				.doCatch(JsonProcessingException.class)
+					.to(direct(ERROR_WRITE_TO_ADISSUE))
+					.process(this::prepareErrorResponse)
+					.marshal(setupJacksonDataFormatFor(getContext(), JsonError.class))
+				.doCatch(HttpOperationFailedException.class)
+					.to(direct(MF_ERROR_ROUTE_ID))
+					.process(this::prepareHttpErrorResponse)
+					.marshal(setupJacksonDataFormatFor(getContext(), JsonError.class))
+				.doCatch(Exception.class)
+					.to(direct(MF_ERROR_ROUTE_ID))
+					.process(this::prepareErrorResponse)
+					.marshal(setupJacksonDataFormatFor(getContext(), JsonError.class))
+				.endDoTry();
+
+		// disable http-endpoint
+		from(direct(DISABLE_RESOURCE_ROUTE_ID))
+				.routeId(DISABLE_RESOURCE_ROUTE_ID)
+				.log("Route invoked!")
+				.process(this::prepareRestAPIContext)
+				.process(this::attachAuthenticateRequest).id(DISABLE_RESOURCE_ATTACH_AUTHENTICATE_REQ_PROCESSOR_ID)
+				.to(direct(REST_API_EXPIRE_TOKEN))
+				.process(this::disableRestAPIProcessor).id(DISABLE_RESOURCE_ROUTE_PROCESSOR_ID)
+				.process(exchange -> this.prepareExternalStatusCreateRequest(exchange, JsonExternalStatus.Inactive)).id(DISABLE_PREPARE_EXTERNAL_STATUS_CREATE_REQ_PROCESSOR_ID)
+				.to("{{" + ExternalSystemCamelConstants.MF_CREATE_EXTERNAL_SYSTEM_STATUS_V2_CAMEL_URI + "}}")
+				.end();
+		//@formatter:on
+	}
+
+	private void prepareResponse(@NonNull final Exchange exchange)
+	{
+		@SuppressWarnings("unchecked") final List<Object> list = exchange.getIn().getBody(List.class);
+		if (list.size() == 1)
```

Comment:
pls verify that the list i not null, 
so if it is, we have a better starting point for the troubleshooting